config/environments/development.rb

```diff
@@ -6,6 +6,9 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
+  # using 3001 to avoid conflict with our other applications
+  Rails.application.routes.default_url_options[:host] = 'localhost:3001'
```

Comment:
🌱  Interesting[ stackoverflow post](https://stackoverflow.com/questions/22691588/understanding-rails-default-url-options) that might allow us to set this value more elegantly

Reply:
🤔 I've read the post a few times and not sure I completely understand the problem it solves, but would appreci8 a zoom explanation if possible 🙏